src/constants.ts

```diff
@@ -2,3 +2,9 @@ export const NODE_ENV = process.env.NODE_ENV;
 export const ENDPOINT = NODE_ENV === 'prod' ?
```

Comment:
Can you also rename this as AINFT_SERVER_ENDPOINT?

Reply:
It looks more clear! done!